api/flask_bot_server.py

```diff
@@ -1,70 +0,0 @@
-from __future__ import absolute_import
-from flask import Flask, request, jsonify
-import os
-import sys
-import json
-from typing import Any, Dict, Mapping, Union, List
-from werkzeug.exceptions import BadRequest
-from six.moves.configparser import SafeConfigParser
```

Comment:
@vabs22: Isn't werkzeug something we might want to specify as a dependency? On that note, do we need it? Why can't we use Flask?